fuel-types/src/array_types.rs

```diff
@@ -346,6 +346,7 @@ key!(Address, 32);
 key!(AssetId, 32);
 key!(BlobId, 32);
 key!(ContractId, 32);
+key!(TxId, 32);
```

Comment:
It would be nice to also add tests for this type(like we have for other types)

Reply:
Yep, will do.